src/ember-app.js

```diff
@@ -26,10 +26,14 @@ class EmberApp {
    * Create a new EmberApp.
    * @param {Object} options
    * @param {string} options.distPath - path to the built Ember application
-   * @param {Sandbox} [options.sandbox=VMSandbox] - sandbox to use
+   * @param {Sandbox} options.sandbox=VMSandbox - sandbox to use
    * @param {Object} [options.sandboxGlobals] - sandbox variables that can be added or used for overrides in the sandbox.
    */
   constructor(options) {
+    if (typeof options.sandbox === 'undefined' || !(options.sandbox instanceof Object)) {
```

Comment:
When this assertion fires, it seems like a good teachable moment. Is there something concise we can say, to get people using public API? Or if its a tad longer, maybe a short markdown file can be provided to docs/ directory?


If that providing additional details here is redundant, should we leave a helpful assertion or is that harmful?